dragonfly/server.go

```diff
@@ -335,7 +335,7 @@ func (server *Server) handleConn(conn *minecraft.Conn) {
 		GameRules:                    map[string]interface{}{"naturalregeneration": false},
 		Difficulty:                   2,
 		Items:                        server.itemEntries(),
-		PlayerMovementSettings:       protocol.PlayerMovementSettings{MovementType: protocol.PlayerMovementModeServer},
+		PlayerMovementSettings:       protocol.PlayerMovementSettings{MovementType: protocol.PlayerMovementModeServer, ServerAuthoritativeBlockBreaking: true},
```

Comment:
Do we actually need this? If we don't use this we don't need any changing of the block breaking.

Reply:
It seems to make block breaking feel a lot less laggy. Even if we reverted this, continue destroying seems to have been changed so it would still need some block breaking changes anyways.

Reply:
I thought the block breaking was fixed in the previous version, so it'd be weird if that broke now. Either way, if it actually is less laggy I suppose we should look into that. I do think the current way isn't going to cut it though.

Reply:
For sure